cmd/nitro/nitro.go

```diff
@@ -7,6 +7,7 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"crypto/rand"
+	encoding_json "encoding/json"
```

Comment:
how about leaving standard package `enconding/json` unaliased and making an alias for `github.com/knadh/koanf/parsers/json` instead  (eg. `koanfjson` as in cmd/util/confighelpers/configuration.go)?

Reply:
done.